terminal_client.go

```diff
@@ -115,6 +115,18 @@ func (c *TerminalClient) listCmd() error {
 	if err != nil {
 		return err
 	}
+
+	deployTable := make(map[string][]*Deploy)
+
+	for _, deploy := range deploys {
+		// If we have a deploy with the given id, merge in data
+		if _, ok := deployTable[deploy.Id]; ok {
```

Comment:
You can just write this:

deployTable[deploy.Id] = append(deployTable[deploy.Id], deploy)

because referencing a non-existent key in Go returns the nil item of that type, in this case a slice, which is the empty slice.